packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -94,8 +98,14 @@ export class PincodeSet extends React.Component<Props, State> {
   onCompletePin1 = () => {
     if (this.isPin1Valid(this.state.pin1)) {
       this.props.navigation.setParams({ isVerifying: true })
+      if (this.props.route.params.changePin) {
```

Comment:
I think you need optional chaining on all of these references to `changePin` because `this.props.route.params` can be undefined

Reply:
thanks!